templates/bake/Template/index.twig

```diff
@@ -65,7 +65,14 @@
                     <td class="actions">
                         <?= $this->Html->link(__('View'), ['action' => 'view', {{ pk|raw }}]) ?>
                         <?= $this->Html->link(__('Edit'), ['action' => 'edit', {{ pk|raw }}]) ?>
-                        <?= $this->Form->postLink(__('Delete'), ['action' => 'delete', {{ pk|raw }}], ['confirm' => __('Are you sure you want to delete # {0}?', {{ pk|raw }})]) ?>
+                        <?= $this->Form->postLink(
```

Comment:
Wouldnt deleteLink() be cleaner to avoid the assoc array config overhead?

Reply:
Been using this for the last decade:
```php
	public function deleteLink(string|HtmlStringable $title, array|string|null $url = null, array $options = []): string {
		$options['method'] = 'delete';
		if (!isset($options['class'])) {
			$options['class'] = 'delete-link';
		}

		return $this->postLink($title, $url, $options);
	}
```

Reply:
We don't have a `deleteLink` method. We could add one though.

Reply:
I too was thinking of adding `deleteLink()`.

Reply:
https://github.com/cakephp/cakephp/pull/18144